image_proc/launch/image_proc.launch.py

```diff
@@ -30,36 +30,74 @@
 # ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE
 # POSSIBILITY OF SUCH DAMAGE.
 
-from launch import LaunchDescription
-from launch_ros.actions import ComposableNodeContainer
+import sys
+
+from launch import LaunchDescription, LaunchIntrospector, LaunchService
+from launch_ros import actions, get_default_launch_description
+
 from launch_ros.descriptions import ComposableNode
 
+# Set namespace for all nodes to /camera (for readability purposes)
+NAMESPACE = "/camera"
+
 
 def generate_launch_description():
-    return LaunchDescription([
-        ComposableNodeContainer(
-            package='rclcpp_components', node_executable='component_container',
-            node_name='image_proc_container', node_namespace='',
-            composable_node_descriptions=[
-                ComposableNode(
-                    package='image_proc',
-                    node_plugin='image_proc::DebayerNode',
-                    node_name='debayer'
-                ),
-                ComposableNode(
-                    package='image_proc',
-                    node_plugin='image_proc::RectifyNode',
-                    node_name='rectify_mono'
-                ),
-                ComposableNode(
-                    package='image_proc',
-                    node_plugin='image_proc::RectifyNode',
-                    node_name='rectify_color',
-                    remappings=[
-                        ('image_mono', 'image_color'),
-                        ('image_rect', 'image_rect_color')
-                    ]
-                ),
-            ],
-        ),
-    ])
+    ld = LaunchDescription()
+
+    # Load composable container
+    image_processing = actions.ComposableNodeContainer(
+        node_name="image_proc_container",
+        node_namespace=NAMESPACE,
+        package='rclcpp_components',
+        node_executable='component_container',
+        composable_node_descriptions=[
+            ComposableNode(
+                package='image_proc',
+                node_plugin='image_proc::DebayerNode',
+                node_name='debayer_node',
+                node_namespace=NAMESPACE,
```

Comment:
Can you verify for me that putting the `node_namespace` as `NAMESPACE` doesn't also namespace the topics? 

E.g. `('/image', NAMESPACE + '/image')` is that necessary with the `node_namespace=NAMESPACE`?

Reply:
I'll check that out tomorrow morning PST and get back to you then :)

Reply:
If it turns out that its redundant, remove the node namespace and leave the remaps.

Reply:
If the topic names do not have a slash (which they shouldn't), then the `node_namespace` argument will namespace the topics.

Reply:
It looks like many (if not all) have leading `/`s.

Reply:
That puts them in the root namespace which is bad juju. From http://wiki.ros.org/Names:

>Names that start with a "/" are global -- they are considered fully resolved. Global names should be avoided as much as possible as they limit code portability.

The "correct" way to handle topic names is to use un-prefixed names and let namespacing handle the full path resolution.

Reply:
Yeah, agreed. I think for this demo the right answer is to remove the `/` and then do the remaps without the node_namespace set.

Reply:
I removed the namespace and changed all the topics to relative